src/main/java/io/opencensus/integration/jdbc/OcWrapCallableStatement.java

```diff
@@ -14,548 +14,554 @@
 
 package io.opencensus.integration.jdbc;
 
+import io.opencensus.common.Scope;
```

Comment:
This is what I was avoiding by encapsulating all those operations in RoundtripTrackingSpan.